static/index.html

```diff
@@ -5,13 +5,41 @@
 </head>
 
 <body>
-  <h1>Install Production Firmware</h1>
-  <esp-web-install-button
-    manifest="https://firmware.esphome.io/home-assistant-voice-pe/home-assistant-voice/manifest.json"></esp-web-install-button>
+  <div>
 
-  <h2>Install Beta Firmware</h2>
-  <esp-web-install-button
-    manifest="https://firmware.esphome.io/home-assistant-voice-pe/home-assistant-voice/manifest-beta.json"></esp-web-install-button>
+    <h1>Install Voice PE Latest Firmware</h1>
+    <esp-web-install-button
+      manifest="https://firmware.esphome.io/home-assistant-voice-pe/home-assistant-voice/manifest.json">
+    </esp-web-install-button>
+  </div>
+  <br />
+  <div>
+    <a id="specific-version">> Install Specific Version</a>
+    <div id="specific-version-input" style="display: none">
+      Version: <input />
+    </div>
+  </div>
```

Comment:
This is a perfect use case for a [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#try_it)

Reply:
Lol, Had no clue about that. Will change

Reply:
Fixed in [`1e8bb9b` (#173)](https://github.com/esphome/home-assistant-voice-pe/pull/173/commits/1e8bb9b6bc76212c71d4222e3c899ea57e1bfa1e)